test/core/end2end/fuzzers/client_fuzzer.cc

```diff
@@ -57,6 +63,13 @@ static void dont_log(gpr_log_func_args* /*args*/) {}
 
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
   if (squelch) gpr_set_log_function(dont_log);
+  grpc_event_engine::experimental::SetEventEngineFactory([]() {
```

Comment:
It's necessary to get this work landed, otherwise we get combiner offloads causing threads to spin up and confusing the logic here. Better to keep it together.